Marlin/Marlin_main.cpp

```diff
@@ -2907,14 +2907,8 @@ inline void gcode_G28() {
           #endif
 
           feedrate = homing_feedrate[Z_AXIS];
-
-          #if HAS_BED_PROBE
-            do_blocking_move_to_z(z_dest);
-          #else
-            line_to_z(z_dest);
-            stepper.synchronize();
-          #endif
-
+          line_to_z(z_dest);
+          stepper.synchronize();
```

Comment:
I figured on saving one function call if not needed. I know it's a very minor thing. In the long run I'm sure we can continue to consolidate all these movement helpers.